app/assets/src/styles/_header.scss

```diff
@@ -142,18 +126,22 @@ a {
   }
 }
 
-.profile-header-dropdown {
-  border: none !important;
+.profile-header-menu.menu {
+  float: right;
   margin-top: 1rem;
   margin-right: 0.5rem;
+  background-color: rgba(0, 0, 0, 0);
+  min-height: 0;
+  box-shadow: none;
+  border: none;
   .text {
-    font-size: 1rem !important;
+    font-family: Open Sans, Helvetica Neue, Arial, Helvetica, sans-serif;
```

Comment:
for font-family could you import themes/_typograpy and then use $font-family-sans-serif

Reply:
whoops i committed before seeing this - will clean up in follow-up diff

Reply:
nm, i see i can do rgba($black, 0)